src/ci/docker/host-x86_64/x86_64-gnu-tools/Dockerfile

```diff
@@ -100,6 +100,6 @@ COPY scripts/shared.sh /scripts/
 # The `browser-ui-test.version` file is also used by bootstrap to emit warnings in case
 # the local version of the package is different than the one used by the CI.
 ENV SCRIPT /tmp/checktools.sh ../x.py && \
-  npm install browser-ui-test@$(head -n 1 /tmp/browser-ui-test.version) --unsafe-perm=true && \
+  npm install --no-package-lock browser-ui-test@$(head -n 1 /tmp/browser-ui-test.version) --unsafe-perm=true && \
```

Comment:
Now that it's in the `package.json` file, do we still need the `head -n 1 /tmp/browser-ui-test.version` part?

Reply:
I'm currently opening a separate PR to refactor how `browser-ui-test` gets installed, since this method seems quite fragile.